resources/css/app/_action-column.css

```
@@ -0,0 +1,91 @@
+        justify-content: end;
+    }
+}
+
+td.action-column > div {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    gap: var(--action-column-button-gap);
+}
+
+@media (width < theme(--breakpoint-lg)) {
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)